lib/web/assistant.go

```diff
@@ -500,6 +504,49 @@ type usageReporter interface {
 	SubmitUsageEvent(ctx context.Context, req *proto.SubmitUsageEventRequest) error
 }
 
+// assistGenAuditQueryLoop reads the user's input and generates an audit query.
+func (h *Handler) assistGenAuditQueryLoop(ctx context.Context, assistClient *assist.Assist, ws *websocket.Conn, username string, usageRep usageReporter) error {
+	for {
+		_, payload, err := ws.ReadMessage()
+		if err != nil {
+			if wsIsClosed(err) {
+				break
+			}
+			return trace.Wrap(err)
+		}
+
+		onMessage := func(kind assist.MessageType, payload []byte, createdTime time.Time) error {
+			return onMessageFn(ws, kind, payload, createdTime)
+		}
+
+		toolCtx := &tools.ToolContext{User: username}
+
+		tokenCount, err := assistClient.RunTool(ctx, onMessage, "Audit Query Generation", string(payload), toolCtx)
```

Comment:
"Audit Query Generation" is the tool name which is a string. Can you create a constant (to prevent typos) and maybe create a new type to not mix the order of the string when calling this function?